core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -158,7 +158,11 @@ object TestUtils extends Logging {
   }
 
   def getBrokerListStrFromServers(servers: Seq[KafkaServer], protocol: SecurityProtocol = SecurityProtocol.PLAINTEXT): String = {
-    servers.map(s => formatAddress(s.config.hostName, s.boundPort(protocol))).mkString(",")
+    servers.map(svr => { 
+      val x = formatAddress(svr.config.hostName, svr.boundPort(protocol))
+      val hname = svr.config.hostName
+      info(s"### Formatted address: $protocol, $hname, $x")
```

Comment:
Would it not be a little nicer if we printed all of the addresses in one go, instead of like this?